footer.php

```diff
@@ -332,3 +334,102 @@ function make_navbutton($imgsrc, $bgcolour, $title, $url, $classes = null, $neww
     }
     return $output;
 }
+
+/**
+ * Create Activity completion buttons
+ *
+ * @return string
+ */
+function make_activitycomplete_button() {
+
+    global $COURSE, $DB, $CFG, $PAGE, $OUTPUT;
+
+    $output = "";
+    $style = "";
+
+    if (!$settings = $DB->get_record('navbuttons', array('course' => $COURSE->id))) {
+        return $output;
+    }
+
+    if(!$settings->completebuttonshow) {
```

Comment:
Minor code layout - could you run this against local_codechecker (but ignore any legacy layout issues that are in the existing code).